includes/frontend/class-wcmp-checkout.php

```diff
@@ -328,8 +328,8 @@ public static function save_delivery_options(int $orderId): void
     {
         $order = WCX::get_order($orderId);
 
-        $shippingMethod       = sanitize_text_field($_POST['shipping_method'][0] ?? '');
-        $highestShippingClass = (int) ($_POST['myparcel_highest_shipping_class'] ?? 0);
+        $shippingMethod       = sanitize_text_field(wp_unslash($_POST['shipping_method'][0] ?? ''));
+        $highestShippingClass = (int) (wp_unslash($_POST['myparcel_highest_shipping_class'] ?? ''));
```

Comment:
nu is 'ie weer niet sanitized 🤡

Reply:
Maar dat hoeft toch ook niet? Het wordt meteen een int, de waarde wordt nooit gebruikt...